docs/guides/Search/how-search-parameters-work.md

```diff
@@ -24,46 +24,51 @@ The URL will always contain the term entered in the search field:
 
 `www.store.com/shoes`
 
-The browsing filters also have specific URLs defined by certain parameters in the URL. The parameters are:
+The browsing filters also have specific URLs defined by certain parameters in the URL, which can follow one of these options:
+
+- [Default 1](#default-1): Valid only for Legacy CMS Portal stores.
+- [Default 2](#default-2): Valid for Legacy CMS Portal and VTEX IO stores. Recommended for best performance results.
 
 ## Default 1
 
+> ⚠️ This option works only for Legacy CMS Portal stores. Refer to [Default 2](#default-2) for VTEX IO stores.
+
 | URL |
 | - |
 | `www.store.com/busca/?fq=C:[DepartmentId/CategoryId/SubcategoryId]&fq=B:[BrandId]&fq=H:[CollectionId]&fq=specfct[ProductFieldId/Sku]:[SearchValue]&ft=[SearchTerm]` |
 
 Where:
 
-- **C:\[DepartmentId/CategoryId/SubcategoryId]**: Shows products of a specific category based on the IDs indicated for the department, category, and subcategory. This code appears next to category names in **Products** > **Catalog** > **Categories**. ![](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/how-search-parameters-work-2.png)
+- **C:\[DepartmentId/CategoryId/SubcategoryId]:** Shows products of a specific category based on the IDs indicated for the department, category, and subcategory. This code appears next to category names in **Products** > **Catalog** > **Categories**. ![](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/how-search-parameters-work-2.png)
 
-> ℹ️ The category ID can also be found on your edit page at the end of the URL.
+  > ℹ️ The category ID can also be found on your edit page at the end of the URL.
 
 - **B:[BrandId]:** Shows products from a specific brand based on the indicated ID. This code is shown at the end of the URL, on the page used for changing the brand, in**Products**>**Catalog**>**Brands**. ![](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/how-search-parameters-work-3.png)
 - **H:[CollectionId]**: Shows products of a specific collection based on the indicated ID. This code is indicated when editing a collection in**Store setup** > **CMS** > **Layout** > **CMS** > **Product Clusters (Collections)**. ![](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/how-search-parameters-work-4.png)
 
-> ⚠️ There are two ways to configure collections: through the CMS or the Collections module (Beta). This article explains how to [configure collections through the CMS](https://help.vtex.com/en/tutorial/adding-collections-cms--2YBy6P6X0NFRpkD2ZBxF6L).
+  > ⚠️ There are two ways to configure collections: through the CMS or the Collections module (Beta). This article explains how to [configure collections through the CMS](https://help.vtex.com/en/tutorial/adding-collections-cms--2YBy6P6X0NFRpkD2ZBxF6L).
```

Comment:
```suggestion
  > ⚠️ There are two ways to configure collections: through the Legacy CMS Portal or the Collections module (Beta). This article explains how to [configure collections through the Legacy CMS Portal](https://help.vtex.com/en/tutorial/adding-collections-cms--2YBy6P6X0NFRpkD2ZBxF6L).
```

Reply:
Fiquei em dúvida com "this article". O artigo atual ou o do link? 
Sugiro algo como:
Refer to [Adding collections CMS](https://help.vtex.com/en/tutorial/adding-collections-cms--2YBy6P6X0NFRpkD2ZBxF6L) for more information on...